### ISSUE TITLE
Crash on make install

### ISSUE THREAD

**Issue by danielravina** [Dridi/libvmod-querystring#15]
System:
Vagrant Ubuntu precise32
Varnish 3.0.7 revision f544cd8

```
root@precise32:~# git https://github.com/Dridi/libvmod-querystring.git ~/libvmod-querystring [Good]
root@precise32:~# cd ~/libvmod-querystring
root@precise32:~/libvmod-querystring# ./autogen.sh # [Good]
root@precise32:~/libvmod-querystring# ./configure VARNISHSRC=/opt/varnish-3.0.7 VMODDIR=/usr/local/lib/varnish/vmods # [Good] 
root@precise32:~/libvmod-querystring# make # [Good]
root@precise32:~/libvmod-querystring# make install # [Bad]
Making install in src
make[1]: Entering directory `/root/libvmod-querystring/src'
/bin/bash ../libtool --tag=CC   --mode=compile gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I.. -I/opt/varnish-3.0.7/include -I/opt/varnish-3.0.7/bin/varnishd -I/opt/varnish-3.0.7   -DVARNISH_MAJOR=3 -DHAVE_VARNISH_3_0_0 -DQS_ENABLE_LOGGING -g -O2 -MT vmod_querystring.lo -MD -MP -MF .deps/vmod_querystring.Tpo -c -o vmod_querystring.lo vmod_querystring.c
libtool: compile:  gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I.. -I/opt/varnish-3.0.7/include -I/opt/varnish-3.0.7/bin/varnishd -I/opt/varnish-3.0.7 -DVARNISH_MAJOR=3 -DHAVE_VARNISH_3_0_0 -DQS_ENABLE_LOGGING -g -O2 -MT vmod_querystring.lo -MD -MP -MF .deps/vmod_querystring.Tpo -c vmod_querystring.c  -fPIC -DPIC -o .libs/vmod_querystring.o
vmod_querystring.c: In function 'is_param_regfiltered':
vmod_querystring.c:284:2: warning: passing argument 1 of 'VRT_re_match' from incompatible pointer type [enabled by default]
/opt/varnish-3.0.7/include/vrt.h:149:5: note: expected 'const struct sess *' but argument is of type 'char *'
vmod_querystring.c:284:2: error: too few arguments to function 'VRT_re_match'
/opt/varnish-3.0.7/include/vrt.h:149:5: note: declared here
make[1]: *** [vmod_querystring.lo] Error 1
make[1]: Leaving directory `/root/libvmod-querystring/src'
make: *** [install-recursive] Error 1
```
Thanks!

**Comment by danielravina** [comment]
Read through the pull requests and copied code for the support for Varnish 3.0.7 - That solved it. Please merge this request. Thanks!

**Comment by Dridi** [comment]
Sorry for missing the pull request, it is done, and I have released a new 0.3 version.
